reactornettybenchmark/pom.xml

```diff
@@ -15,8 +15,25 @@
     <properties>
         <reactornetty.version>0.8.3.RELEASE</reactornetty.version>
         <reactor.version>3.2.2.RELEASE</reactor.version>
+        <!--
+        <reactornetty.version>0.8.4.BUILD-SNAPSHOT</reactornetty.version>
```

Comment:
Should we make the snapshot the default and comment out release version?

Reply:
If you OK with dealing with instability with SNAPSHOT version. IMO, lets keep this as stable version and toggle to SNAPSHOT if (rxnetty issues claims certain issues are fixed) there is a need.